### PR TITLE
IA-2 (12)

### PR DESCRIPTION
**Pull request by kevindoingbizas**

### CODE REVIEW

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 1]
```diff
@@ -147,12 +147,16 @@
 - control_key: IA-2 (12)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: complete
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
  implementation_status: not applicable
```

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 2]
```diff
@@ -147,12 +147,16 @@
...
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-317
+        OCP offloads (via: https://access.redhat.com/documentation/en-us/
+        openshift_container_platform/4.3/html/authentication/
+        understanding-identity-provider)
+        responsibility for authentication, session management, and OAuth
+        controls to the hosting environment’s LDAP and leverages CAC/PKI,
+        FIPS 140-2 compliant ciphers, SSO and does not provide for group
+        accounts.
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]:
```suggestion
        Acceptance and electronic verification of Personal
        Identity Verification (PIV) credentials is the responsibility
        of the organizational identity provider (such as
        LDAP, Active Directory, Red Hat SSO, etc).

        To aid system administrators, documentation on
        supported authentication providers can be found in the
        OpenShift product documentation:
        https://access.redhat.com/documentation/en-us/openshift_container_platform/latest/html/authentication/understanding-identity-provider

```

Comment (kevindoingbizas) [reply openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]: I like the additional context

Comment (shawndwells) [reply openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]: Ok, merging this suggestion into the PR. What about flipping the implementation_status to ``not applicable``?

**Comment by redhatrises** [comment]
The comments in this PR haven't been addressed or resolved in a few months. This PR looks frozen. Closing. Please re-open this PR when the comments are addressed appropriately.

**Pull request closed without merging**